src/main/java/bio/terra/janitor/db/JanitorDao.java

```diff
@@ -75,24 +70,21 @@ public String createResource(
               .map(
                   entry ->
                       new MapSqlParameterSource()
-                          .addValue("tracked_resource_id", trackedResourceId.getUUID())
+                          .addValue("tracked_resource_id", trackedResourceId.id())
                           .addValue("key", entry.getKey())
                           .addValue("value", entry.getValue()))
               .toArray(size -> new MapSqlParameterSource[labels.size()]);
-      try {
-        jdbcTemplate.batchUpdate(insertLabelSql, sqlParameterSourceList);
-      } catch (DuplicateKeyException e) {
-        throw new DuplicateDbKeyException(
-            "Duplicate label found for tracked_resource_id: "
-                + trackedResourceId
-                + " already exists.",
-            e);
-      }
+
+      jdbcTemplate.batchUpdate(insertLabelSql, sqlParameterSourceList);
     }
-    return trackedResourceId.toString();
+    return trackedResourceId;
   }
 
   @VisibleForTesting
```

Comment:
nit: move annotation below javadoc.

Reply:
but keep some of the comment? It's still important to not mess with this for postgres.